### DISCUSSION TITLE
Find all resources (of any type) with certain labels

### DISCUSSION THREAD

**Discussion by SoerenHenning** [fabric8io/kubernetes-client#4021] [Q&A] [answered]
Hi, is there any way to find (and eventually delete) all resources with some specified labels, independent of the resource type? The expected return type would be something such as a list of `HasMetadata` objects.

**Comment by shawkins** [comment, marked as answer]
This has been asked for a couple of times, but has not been implemented: #887

Based upon the support that is now available in the client, if you don't want to explicitly enumerate the types you are searching over, you would have to walk the apiGroup information to find all kinds known to the api server.

**Reply by SoerenHenning** [reply]
Thank you, I probably was a bit unlucky with my search terms.
